packages/replay-internal/src/types/performance.ts

```diff
@@ -110,11 +110,11 @@ export interface WebVitalData {
   /**
    * The recording id of the web vital nodes. -1 if not found
    */
-  nodeIds?: number[];
+  nodeIds: number[] | undefined;
   /**
    * The layout shifts of a CLS metric
    */
-  attributions?: { value: number; nodeIds?: number[] }[];
```

Comment:
I think we can leave the types on the top level nodeIds and attributions... it's valid for these to not exist at all for web vitals, where as the nodeIds inside of attributions should always exist if we have a value, but undefined is a valid value for it if we can't find the node id in the DOM.